aiinfra-cluster/modules/gke-cluster/main.tf

```diff
@@ -133,9 +133,10 @@ resource "google_container_cluster" "gke-cluster" {
   # Allow planned maintenance during this time period.
   maintenance_policy {
     recurring_window {
-      start_time = "2019-01-01T09:00:00Z"
-      end_time = "2019-01-01T17:00:00Z"
-      recurrence = "FREQ=WEEKLY;BYDAY=MO,TU,WE,TH,FR"
+      start_time = "2023-01-01T09:00:00Z"
```

Comment:
Can we base it on current date? The goal is "6 months from the moment of creation", which is another way to say "never" if cluster gets recreated before then.

Reply:
The way i've seen people using this is to have a big time window and use the recurrence to control frequency.  Using something like datetime.now will change every time you run the tool even for the same GKE cluster and it will end up changing the cluster.